sp800-63b/sec1_2_introduction.md

```diff
@@ -4,29 +4,29 @@
 
 _This section is informative._
 
-This document and its companion documents, [SP 800-63-3](sp800-63-3.html), [SP 800-63A](sp800-63a.html), and [SP 800-63C](sp800-63c.html), provide technical guidelines to agencies for the implementation of digital authentication.
+This document and its companion documents, [Special Publication (SP) 800-63-3](sp800-63-3.html), [SP 800-63A](sp800-63a.html), and [SP 800-63C](sp800-63c.html), provide technical guidelines to agencies for the implementation of digital authentication.
```

Comment:
Later sections use double brackets when linking to other Special Publications. I'm not sure whether we want to show the brackets to make it clear that they're references like that, but in any case we should be consistent.